tox-check-i18ndude-command.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+# Shell script to test the i18ndude command in tox.
+# In tox commands you cannot use shell constructs like double pipes.
```

Comment:
Should we move this in the test folder?

Reply:
That could make things easier. Let me try.